index.android.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+import { NativeModules } from 'react-native';
+import type { Region } from './models/Location';
+import type { Callback } from './models/Callback';
+import debounce from './utils/debounce';
+
+const { RNReverseGeocode } = NativeModules;
+
+const searchForLocations = (
+  searchText: string,
+  region: Region,
+  callback: Callback,
+  debounceMs: number = 200
+) => {
+  debounce(
+    RNReverseGeocode.searchForLocations(searchText, region, callback),
+    debounceMs
+  );
+};
+
+const isPresent = (
+  callback: Callback,
```

Comment:
I believe the Callback type should be more generic, because the current type is 
```
export type Callback = (err: string, res: Result) => void;
```
and isPresent returning just a boolean does not respect that signature.